src/removeOldFiles.js

```diff
@@ -0,0 +1,29 @@
+const { cozyClient, log } = require('cozy-konnector-libs')
+const bluebird = require('bluebird')
+
+const oldNameFormat = /\d{4}\d{2}_mgen\.pdf/
+const isOldFile = file => {
+  return oldNameFormat.exec(file.attributes.name)
+}
+
+const lsDir = async folder => {
+  const limit = 100000
+  const folderId = folder._id
+  const data = await cozyClient.fetchJSON('GET', `/files/${folderId}?page[limit]=${limit}`, null, {
+    processJSONAPI: false
+  })
+  return data.included
+}
+
+const removeOldFiles = async fields => {
```

Comment:
`removeOldFiles` may remove files which are too old and will not be fetched in the new format. We should check the list of fetched bills and not remove files older than the oldest fetched bill